### PR TITLE
New Icons

### PR DESCRIPTION
**Pull request by REJack**
PR for #4

### CODE REVIEW

**Comment by twolfson** [comment]
Woot, thanks for the work on this =)

Unfortunately, I've got some changes to request as well as things to discuss:

- Can we remove the Chrome/Safari edits? If we ever have to pull from `upstream`, it will cause unnecessary work as we don't plan on maintaining those in this fork
- It looks like we're planning on compiling 2 extensions. This is a bit more annoying for me as a maintainer. Is there any way to consolidate the 2 extensions?
    - e.g. detect what the current Firefox theme is and toggle icons at load time?
    - What do other extensions do? Are there any that do detection? Do most of them have 2 variations published?
    - Maybe Firefox itself has some way to handle this for its own icons? Can we find the relevant source code?

**Comment by REJack** [comment]
1. Sure i can remove the Chrome/Safari changes 😄.
2. About the additional extension
   - It's not able to detect the current Firefox theme, theme.getCurrent() doesn't work with the builtin themes(lightweightThemes), only for custom themes.
   - I've tested also to change the disabled SVG Icon with `fill="context-fill"` but I didn't got it working :/ (It works only for `@mozilla-` & `@testpilot-` extensions).
   - After some researches now I've found [this thread](https://discourse.mozilla.org/t/add-on-svg-toolbar-icon-how-to-fit-the-color-scheme-of-the-current-theme/18497) and the way to handle it only with one extension, I'll change the addon later and remove the additional firefox extension.

It's ok, if I create a complete new PR without the Chrome/Safari edits?

**Comment by twolfson** [comment]
Yea, feel free to update this PR or open a new one. I don't think anything should block you from updating this branch though:

```bash
# Copy files from another branch
git checkout master -- Chrome/ LiveReload.safariextension/
```

Also, do we need to store `src/icons` if they're all in Firefox anyway?

```bash
rm -r src/icons
```

**Comment by REJack** [comment]
It works 😄, but it need at the moment a workaround in about:config `svg.context-properties.content.enabled` to `true`.

**Comment by twolfson** [comment]
Oooh nice! Will the workaround eventually be unnecessary in future versions of Firefox?

**Comment by REJack** [comment]
First Happy New,

I‘ll research this Tomorrow :) I has read some stuff about it, that it should be add already added in 57.0 :/. 


Mit freundlichen Grüßen
Raphael Jackstadt

Von meinem iPhone gesendet

**Comment by REJack** [comment]
@twolfson I found a solution that works without any config tweak, etc.

File: .gitignore
Comment (twolfson) [inline .gitignore]: We can prob roll back this line but I can do that via commandeer

**Review by twolfson — APPROVED** [review]
Great! This all looks fantastic. My schedule is a bit of a mess but I'll land it by the end of the week 👍

File: .gitignore
Comment (REJack) [reply .gitignore]: Should I remove this line?

Comment (twolfson) [reply .gitignore]: Yea, if you have the time. Otherwise, I'll do it before landing. I think we no longer have a `LiveReload-Dark` folder

**Comment by twolfson** [comment]
Trying this out now

**Comment by twolfson** [comment]
I've been seeing this behavior caused by the `null` path but the docs seem to agree that's the way to get the default icon back. I'm realizing I'm on Firefox@58 when 59 is out so I'm going to try upgrading

selection_544

**Comment by twolfson** [comment]
Yep, it was the older Firefox version. Definitely riding the edge here but I think it's fine =)

**Comment by twolfson** [comment]
This has been landed and submitted to https://addons.mozilla.org/. Thanks for all your work on this @REJack :100:

**Pull request merged**
